<launch>

<node pkg = "gps_common" type ="utm_odometry_node" name ="gps_common"/>

<node pkg = "advanced_navigation_driver" type ="advanced_navigation_driver" name = "advanced_navigation_driver" args = "/dev/ttyUSB0 1000000">
<remap from = "/an_device/NavSatFix" to = "/fix"/>


</node>
</launch>